input/signdescriptor.go

```diff
@@ -56,8 +56,9 @@ type SignDescriptor struct {
 	DoubleTweak *btcec.PrivateKey
 
 	// WitnessScript is the full script required to properly redeem the
-	// output. This field will only be populated if a p2wsh or a p2sh
-	// output is being signed.
+	// output. This field should be set to the full script if a p2wsh
+	// output is being signed. For p2wkh it should be set to the hashed
```

Comment:
okay, i see the comment is updated here, squash?

Reply:
Fixed: https://github.com/lightningnetwork/lnd/pull/3829/commits/95cac3f2e74a18bdeb664f2bb043a0b21e2bd266